Makefile

```
@@ -2,16 +2,15 @@ SHELL := /bin/sh
-MVN ?= mvn
-WRO4J_SKIP ?= true
+GRADLEW ?= ./gradlew
 
 # Migration args (override via: make migrate-books MIGRATE_MAX=100 MIGRATE_SKIP=0 MIGRATE_PREFIX=books/v1/ MIGRATE_DEBUG=true)
 MIGRATE_MAX ?= 0
 MIGRATE_SKIP ?= 0
 MIGRATE_PREFIX ?= books/v1/
 MIGRATE_DEBUG ?= false
 
-.PHONY: run build test kill-port migrate-books migrate-books-spring cluster-books
+.PHONY: run build test lint kill-port migrate-books migrate-books-spring cluster-books check-s3-in-db
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)